contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -1982,11 +1990,29 @@ is_rolemember(PG_FUNCTION_ARGS)
 	db_owner_oid = get_role_oid(db_owner_name, false);
 	dbo_role_oid = get_role_oid(dbo_role_name, false);
 	if ((principal_oid == db_owner_oid) || (principal_oid == dbo_role_oid))
+	{
+		if(db_owner_name)
```

Comment:
We can free these before the if. That way we don't need to duplicate it for every if here.

Reply:
Done